README.rst

```diff
@@ -120,7 +173,7 @@ function. Below is an example:
         }
 
     dummy_processor.lti_xblock_default_params = {
-        'custom_course_id': '',
+        custom_course_id': '',
```

Comment:
is this dropped quote intentional?

Reply:
No, I'll put it back it. Thanks for catching that.

Reply:
@marcotuts Fixed.